ui/src/environments/environment-type.ts

```diff
@@ -69,8 +69,8 @@ export interface Environment extends EnvironmentBase {
   // Indicates that the current server is a local server where client-side
   // debugging should be enabled (e.g. console.log, or devtools APIs).
   debug: boolean;
+
```

Comment:
silly UI change to trigger e2e tests.  See https://precisionmedicineinitiative.atlassian.net/browse/RW-10039

Reply:
I've done this too. If you like, you can revert it once the tests have passed and add a comment declaring the results.

Reply:
This is partially a reversion of a similar change made in the other PR :)